### PR TITLE
Extending update

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| Versions      | master

### CODE REVIEW

File: docs/guide/extending/extending_ez_platform.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ See [Extending menus](extending_menus.md) for an example on how to use an event
 to extend one of the menus in the Back Office.
 
 Some extensibility, such as [adding custom tags](extending_online_editor.md#custom-tags),
-is possible without writing your own code, with configuration and templating only.
+is possible without writing your own code, with a configuration and templating only.
```
Comment (DominikaK) [inline docs/guide/extending/extending_ez_platform.md]: Beg to differ, I don't think configuration is countable in this case

File: docs/guide/extending/extending_multifile_upload.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ Without all the following properties the Multi-file Upload will not work.
 
 ### Optional properties
 
-Optionally, the Multi-file Upload module can take a following list of props:
+Optionally, the Multi-file Upload module can take a following list of properties:
```
Comment (DominikaK) [inline docs/guide/extending/extending_multifile_upload.md]: "prop" is a specific term in React, see https://reactjs.org/docs/components-and-props.html

File: docs/guide/extending/extending_ez_platform.md (hunk 1 above)
Comment (MagdalenaZuba) [inline docs/guide/extending/extending_ez_platform.md]:
```suggestion
is possible without writing your own code, with configuration and templating only.
```

File: docs/guide/extending/extending_multifile_upload.md (hunk 2 above)
Comment (MagdalenaZuba) [inline docs/guide/extending/extending_multifile_upload.md]:
```suggestion
Optionally, the Multi-file Upload module can take a following list of prop:
```

**Pull request merged**